src/openrct2/windows/install_track.c

```diff
@@ -261,7 +261,8 @@ static void window_install_track_paint(rct_window *w, rct_drawpixelinfo *dpi)
     gfx_draw_string_left(dpi, STR_TRACK_DESIGN_NAME, &_trackName, COLOUR_BLACK, x-1, y);
     y += 10;
 
-    rct_string_id friendlyTrackName = RideNaming[td6->type].name;
+    sint32 groupIndex = object_manager_get_loaded_object_entry_index(object_manager_load_object(&td6->vehicle_object));
```

Comment:
There is no protection against NULL here. If one loads a track design that uses a vehicle the user does not have, this could crash the game.